addons/web/static/src/views/fields/priority/priority_field.js

```diff
@@ -66,9 +66,16 @@ export class PriorityField extends Component {
     onStarClicked(value) {
         if (this.props.value === value) {
             this.state.index = -1;
-            this.props.update(this.options[0][0]);
+            this.updateRecord(this.options[0][0]);
         } else {
-            this.props.update(value);
+            this.updateRecord(value);
+        }
+    }
+
+    async updateRecord(value) {
+        await this.props.record.update({ [this.props.name]: value });
+        if (this.props.readonly) {
```

Comment:
unless readonly really means readonly lol. But as mentionned above, maybe we want to save all the time (and I guess we don't get here is the field is really readonly)